sixtyfps_runtime/rendering_backends/gl/event_loop.rs

```diff
@@ -170,6 +175,181 @@ impl std::fmt::Debug for CustomEvent {
     }
 }
 
+fn redraw_all_windows() {
+    let all_windows_weak =
+        ALL_WINDOWS.with(|windows| windows.borrow().values().cloned().collect::<Vec<_>>());
+    for window_weak in all_windows_weak {
+        if let Some(window) = window_weak.upgrade() {
+            window.request_redraw();
+        }
+    }
+}
+
+fn process_window_event(
+    window: Rc<crate::graphics_window::GraphicsWindow>,
+    event: WindowEvent,
+    quit_behavior: sixtyfps_corelib::backend::EventLoopQuitBehavior,
+    control_flow: &mut winit::event_loop::ControlFlow,
+    cursor_pos: &mut Point,
+    pressed: &mut bool,
+) {
+    corelib::animations::update_animations();
```

Comment:
Do we really want to update animation for every single event?
What is the motivation?

Reply:
Good point. That was a little over-eager, we previously only did that for input events. I'll push a change on top to restore that and then it's also a bit more visible where that was/is.